keyboards/mschwingen/modelm/config.h

```diff
@@ -0,0 +1,92 @@
+/*
+ * Copyright 2020 Michael Schwingen
+
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License as published by
+ * the Free Software Foundation, either version 2 of the License, or
+ * (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ */
+
+#pragma once
+
+#include "config_common.h"
+
+/* USB Device descriptor parameter */
+#define VENDOR_ID       0xFEED
+#define PRODUCT_ID      0x0000
```

Comment:
Out of curiosity: Is there documentation on what IDs to use?

Reply:
I don't believe we have this documented yet, but there's an ongoing effort to have all QMK keyboards use a unique `VENDOR_ID`/`PRODUCT_ID` pairing. Since most QMK keyboards use `0xFEED` for Vendor ID, we've been using the shell commands I posted to generate unique Product IDs. (Hash generators work fairly well for this purpose.) Mainly we care that the Vendor ID isn't officially assigned by the [USB IF](https://www.usb.org/ "USB Implementers Forum") (unless the Vendor is using their own officially-assigned ID). We're handling this on a case-by-case basis at the moment, because most contributors change these values voluntarily.

Reply:
Thanks, I was just curious about the process (and if I missed part of the documentation).